frontend/src/lib/api.ts

```diff
@@ -2655,6 +2659,32 @@ const api = {
         })
     },
 
+    async queryAsync<T extends Record<string, any> = QuerySchema>(
```

Comment:
Seems like this could be just an argument on the `query` function above, since the only difference is what function the call is made (`const func = arg ? ApiRequest().queryAsync().create : ApiRequest().query().create`)

Reply:
agreed but typing doesn't like it